appinventor/components-ios/src/ShowAlert.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ShowAlert.swift
+//  AIComponentKit
+//
+//  Created by Dhruv Shrivastava on 05/04/23.
+//  Copyright © 2023 Massachusetts Institute of Technology. All rights reserved.
+//
+
+import Foundation
+
+func showAlert(message: String?) {
```

Comment:
Sir, I think it serves the purpose as if we provide nil it displays the default message and if we provide the message it shows the same message. Also, it may end in code repetition as we have to provide the same default message every time.
I you want I can make it non-optional.

Reply:
We could make it non-optional and specify the default in the signature.